app/views/performance_reports/index.html.slim

```diff
@@ -0,0 +1,40 @@
+h1= t('performance_reports', count: 2)
+
+nav.navbar.section-navigation
+  hr
+  ul.list-inline
+    li= button_link t('.new_report'), new_performance_report_path, 'success', dimension: 'sm'
+  hr
+
+.table-responsive
+  table.table.table-striped
+    thead
+      tr
+        th= t_attr(:created_at)
+        th= t_attr(:period)
+        th= t_attr(:scope)
+        th= t_attr(:extern)
+        th colspan='3'
+
+    tbody
+      - @performance_reports.each do |performance_report|
+        tr
+          td= l(performance_report.created_at.localtime, format: :default)
```

Comment:
Is `.localtime` necessary here? Maybe we should [set `config.time_zone` instead](http://guides.rubyonrails.org/configuring.html#configuring-rails-components)